misc/freetype/imgui_freetype.cpp

```diff
@@ -208,7 +208,9 @@ namespace
     const FT_Bitmap* FreeTypeFont::RenderGlyphAndGetInfo(GlyphInfo* out_glyph_info)
     {
         FT_GlyphSlot slot = Face->glyph;
-        FT_Error error = FT_Render_Glyph(slot, FT_RENDER_MODE_NORMAL);
+        bool monochrome = UserFlags & ImGuiFreeType::Monochrome;
```

Comment:
This can be factored out to `InitFont`:
```cpp
        if (UserFlags & ImGuiFreeType::Monochrome)
            RenderMode = FT_RENDER_MODE_MONO;
        else
            RenderMode = FT_RENDER_MODE_NORMAL;
```
and used here directly:
```cpp
FT_Error error = FT_Render_Glyph(slot, RenderMode);
```

Reply:
Thanks for this PR. One thing less for me to maintain thank's to that change.

Reply:
I agree, this way it's consistent with `LoadFlags`. Changed the code.